nearley/cubeFilters.ne

```diff
@@ -49,10 +50,39 @@ categoryCondition -> ("category"i) (prefixCategoryOpValue | overrideCategoryOpVa
 
 tagsCondition -> ("t"i | "tag"i | "tags"i) stringSetElementOpValue {% ([, condition]) => genericCondition('tags', condition) %}
 
-prefixCategoryOpValue -> ":" prefixCategoryValue {% ([, prefix]) => genericCondition('categoryPrefixes', { $regex: prefix, $options: 'i' }) %}
+@{%
+const categoryConditionFunc = (property, category, value) => ({ 
+  query: {
+    $or: [
+      {
+        $and: [
+          { overrideCategory:true },
+          category,
```

Comment:
shouldn't this be: `{category}` ?

Reply:
No because the query was already built in the caller.